ai_edge_torch/generative/layers/unet/blocks_2d.py

```diff
@@ -294,7 +297,7 @@ class TransformerBlock2D(nn.Module):
       hidden_states
   """
 
-  def __init__(self, config: unet_cfg.TransformerBlock2DConfig):
+  def __init__(self, config: unet_cfg.TransformerBlock2DConfig, dim_overwrite=None):
```

Comment:
what is `dim_overwrite` and could it be added to the config?
if not, please add args comment for it.

Reply:
The idea is to provide ability to override value in config (since the config is shared across different layers).
Comment was added